Detections/SecurityEvent/MultipleFailedFollowedBySuccess.yaml

```diff
@@ -45,4 +45,6 @@ query: |
     | project-away SessionStartedUtc, list_Outcome 
     // where the number of failures before the success is above the threshold 
     | where FailureCountBeforeSuccess >= authenticationThreshold
-    | extend timestamp=StartTime, CustomAccountEntity=Account
+    // expand out ip and computer for customer entity assignment
+    | mvexpand IpAddress=set_IpAddress, Computer=set_Computer
```

Comment:
We should cast to a string otherwise it causes issues with the Rule Creation and generally don't want these to be of dynamic type. Add this line  - | extend IpAddress = tostring(IpAddress), Computer = tostring(Computer)